lib/cartopy/tests/test_img_transform.py

```diff
@@ -94,10 +94,10 @@ def test_griding_data_outside_projection():
 
     # The expected image. n.b. on a map the data is reversed in the y axis.
     expected = np.array(
-        [[3, 3, 3, 3, 3, 3, 3, 3],
-         [3, 3, 3, 3, 3, 1, 2, 2],
-         [2, 2, 3, 1, 1, 1, 1, 2],
-         [1, 1, 1, 1, 1, 1, 1, 1]], dtype=np.float64)
+        [[3, 3, 3, 3, 3, 2, 2, 2],
```

Comment:
I think so, but it would be good if you can double check my logic/math on that. I believe these new numbers are the "closest" from an interpolation standpoint.